pyproject.toml

```
@@ -0,0 +1,24 @@
+[project]
+name = "py-allotax"
+version = "0.1.0"
+description = "Default template for PDM package"
+authors = [
+    {name = "Computational StoryLab", email = "None"},
```

```suggestion
    {name = "Computational StoryLab", email = "peter.dodds@uvm.edu"},
```
I'd put Peter so it won't get outdated.